control_frequency: 20.0
mpc_steps: 20.0
ref_cte: 0.01
ref_etheta: 0.01
ref_vel: 0.01
w_cte: 1000.0
w_etheta: 1000.0
w_vel: 100.0
w_angvel: 100.0
w_accel: 200
w_angvel_d: 0.0
w_accel_d: 10.0
max_angvel: 0.05
max_throttle: 0.5
bound_value: 1.0e3
path_length: 1.5
max_linear_speed: -0.05
min_linear_speed: -0.05
xy_tolerance: 0.015
yaw_tolerance: 0.03
